v23.1/migration-overview.md

```diff
@@ -7,43 +7,40 @@ docs_area: migrate
 1. [Move your data into CockroachDB.](#step-2-move-your-data-to-cockroachdb)
 1. [Test and update your application.](#step-3-test-and-update-your-application)
+{% comment %}
+1. Cutover to production.
+{% endcomment %}
+
+The following MOLT (Migrate Off Legacy Technology) tools simplify migration:
+
+- [Schema Conversion Tool](../cockroachcloud/migrations-page.html).
+- Support for [AWS DMS](aws-dms.html).
 
 {{site.data.alerts.callout_info}}
 If you need to migrate data from a {{ site.data.products.serverless }} cluster to a {{ site.data.products.dedicated }} cluster, see [Migrate data from Serverless to Dedicated](../cockroachcloud/migrate-from-serverless-to-dedicated.html).
 {{site.data.alerts.end}}
 
-## Step 1. Test and update your schema
-
-To begin a new migration to CockroachDB, extract the [data definition language (DDL)](sql-statements.html#data-definition-statements) of the source database. We strongly recommend migrating your database schema to a new CockroachDB database before migrating the data.
-
-You will likely need to update your schema by converting the data definition statements to CockroachDB-compatible statements. This can be due to:
-
-- [Unimplemented features.](#unimplemented-features)
-- [Differences from other databases.](#differences-from-other-databases)
-
-If you are migrating from PostgreSQL, MySQL, Oracle, or Microsoft SQL Server, use the CockroachDB MOLT (Migrate Off Legacy Technology) tools:
+## Step 1. Convert your schema
 
-- [Use the **Schema Conversion Tool**](../cockroachcloud/migrations-page.html) on the {{ site.data.products.db }} Console to analyze your schema for SQL incompatibilities. The tool will identify and help you resolve errors in your schema, and then create a new CockroachDB database with the converted schema.
-- [Move your data](#step-2-move-your-data-to-cockroachdb) to the new database using [AWS DMS](aws-dms.html).
+To begin a new migration to CockroachDB, convert your database schema to an equivalent CockroachDB schema.
 
-### Unimplemented features
-
-CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and is largely compatible with PostgreSQL syntax. However, the following PostgreSQL features do not yet exist in CockroachDB:
+1. Use the source database's tooling to extract the [data definition language (DDL)](sql-statements.html#data-definition-statements) to a `.sql` file.
+1. Upload the `.sql` file to the [**Schema Conversion Tool**](../cockroachcloud/migrations-page.html) on the {{ site.data.products.db }} Console. The tool will convert the syntax, identify [unimplemented features](#unimplemented-features) in the schema, and suggest edits according to CockroachDB [best practices](#schema-design-best-practices).
+	{{site.data.alerts.callout_info}}
+	The Schema Conversion Tool currently accepts `.sql` files from PostgreSQL, MySQL, Oracle, and Microsoft SQL Server.
```

Comment:
The Schema Conversion Tool currently accepts `.sql`... --> The Schema Conversion Tool accepts `.sql`...